git-repo-prep/static/repo-settings/.github/settings.yml

```diff
@@ -0,0 +1,27 @@
+# These settings are synced to GitHub by https://probot.github.io/apps/settings/ via https://github.com/openstax/staxly
+
+repository:
+  # See https://docs.github.com/en/rest/reference/repos#update-a-repository for all available settings.
+  private: false
+
+  has_issues: false
+  has_wiki: false
+  has_projects: false
+  has_downloads: false
+
+  default_branch: main
+
+  allow_auto_merge: true
```

Comment:
Do these need to be included? I thought they were all true by default when a new repo is created.

Similarly, I think the default_branch is main when a new repo is created, no?

Reply:
`allow_auto_merge` has a default value of `false` on private repositories. `delete_branch_on_merge` has a default value of `false` regardless of repository visibility.

`allow_forking`, however, is [completely irrelevant](https://docs.github.com/en/enterprise-server@3.5/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/managing-the-forking-policy-for-your-repository) to public repositories. I removed it from the settings.